seyik/app.js

```diff
@@ -111,6 +110,58 @@ app.post("/posts/register", async (req, res) => {
   res.status(201).json({ message: "postCreated" });
 });
 
+app.patch("/posts", async (req, res) => {
+  const { postingContent, user_id } = req.body;
+
+  await appDataSource.query(
+    `UPDATE posts
+    SET
+    content = ?
+    WHERE posts.user_id = ? 
+    `,
+    [postingContent, user_id]
+  );
+  const rows = await appDataSource.query(
+    `SELECT 
+      users.id as userId,
+      users.name as userName,
+      posts.id as postingId,
+      posts.title as postingTitle,
+      posts.content as postingContent
+      FROM users LEFT JOIN posts
+      ON posts.user_id = users.id
+  `
+  );
+  return res.status(201).json({ data: rows });
+});
+
+app.delete("/posts/:postId", async (req, res) => {
+  const { postId } = req.params;
+  await appDataSource.query(
+    `DELETE
+    FROM posts
+    WHERE posts.id = ?
+    `,
+    [postId]
+  );
+  res.status(200).send();
```

Comment:
```suggestion
  res.status(204).send();
```
204 status code입니다. 200은 response로 body를 담아서 보낼 수 있기 때문에, send()로 빈 응답을 보낼 필요가전혀 없습니다. 이전 리뷰 다시 확인해주세요!